datafusion/expr-common/src/signature.rs

```diff
@@ -120,6 +120,7 @@ pub enum TypeSignature {
     /// Fixed number of arguments of numeric types.
     /// See <https://docs.rs/arrow/latest/arrow/datatypes/enum.DataType.html#method.is_numeric> to know which type is considered numeric
     Numeric(usize),
+    Float(usize),
```

Comment:
One minor comment is perhaps we should call this `Float64` to be explicit that the argument will be 64 bit (rather than maybe f32 or maybe f64)

Reply:
I'm concern that others will add Float32, Float64, Float16... Another idea I'm working on is introducing `TypeSignature::Coercible(Vec<DataType>)`, similar to Exact but try to coerce to target type.